bin/test_all_versions

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+test_version() {
+  echo
+  echo "----------------------------------------------------------------------------------------------------------------------"
+  echo -e "\033[1;34mTesting against ActiveRecord ${1}\033[0m"
+
+  local gemfile="gemfiles/activerecord-${1}.Gemfile"
+  shift
+
+  bundle install --gemfile "${gemfile}" --quiet
+  bundle exec --gemfile "${gemfile}" rspec  "$@"
+}
+
+test_version "5.0"
```

Comment:
This can be a bit smarter perhaps 🔍 but for now this is fine I imagine.

Reply:
do you have a suggestion?  at the moment it solves the problem of testing against all the Gemfiles so I'm happy with it

Reply:
I was just thinking about listing all the `ActiveRecord` versions dynamically? 🤔 . So  you could do a:

```bash
for version in $(...some_command_to_list_versions...); do;
  test_version $version
done;

Reply:
hmm maybe I could just enumerate the gemfiles, lemme have a look

Reply:
thanks for the suggestion :) 

updated!